009-ui-and-admin-server-security.md

```diff
@@ -0,0 +1,84 @@
+# Security Extension
+
+This proposal is supplementary to the security documentation in the [Using Strimzi](https://strimzi.io/docs/operators/latest/using.html#security-str) guide.
+
+## Problem description
+Strimzi currently supports SCRAM-SHA-512, Oauth2 and Mutual TLS for authentication of clients connecting to the Kafka brokers.
+The Strimzi cluster operator can be configured to support all 3 mechanisms for authentication for clients inside or outside the Kubernetes cluster.
+The SCRAM-SHA and Mutual TLS support uses the native support in Kafka for authentication and for Oauth2, authentication is delegated to [keyCloak](https://www.keycloak.org/). 
+
+Authorization is available through native Kafka ACL support with the use of the Kafka `simpleACLAuthorizer`.
```

Comment:
```suggestion
Authorization is available through native Kafka ACL support with the use of the Kafka `SimpleACLAuthorizer`.
```

Reply:
I have removed all references to the SimpleAclAuthorizer.